synapse/handlers/saml_handler.py

```diff
@@ -428,7 +432,7 @@ def parse_config(config: dict) -> SamlConfig:
         return SamlConfig(mxid_source_attribute, mxid_mapper)
 
     @staticmethod
-    def get_saml_attributes(config: SamlConfig) -> Tuple[set, set]:
+    def get_saml_attributes(config: SamlConfig) -> Tuple[Set[str], Set[str]]:
```

Comment:
Can you please remove the return type from the docstring as it's not needed anymore (was probably left here by mistake)?

Reply:
Oops, thanks!